consensus/scheme/rolldpos/roundcalculator.go

```diff
@@ -266,3 +268,36 @@ func (c *roundCalculator) calculateProposer(
 	proposer = delegates[idx%numDelegates]
 	return
 }
+
+type blockTime struct {
+	bc blockchain.Blockchain
+}
```

Comment:
make this struct implement `ChainManager` interface

Reply:
fixed